gradle/versions.gradle

```diff
@@ -0,0 +1,39 @@
+// Variables for entire project
+ext {
+  // Setup
+  minSdkVersion = 14
+  targetSdkVersion = 25
+  compileSdkVersion = 25
+  buildToolsVersion = "25.0.0"
+
+  // Common versions
+  supportLibraryVersion = "25.0.0"
+}
+
+// Plugins
+ext.plugs = [
+    "gradle"                  : "com.android.tools.build:gradle:2.2.3",
+    "androidMavenGradlePlugin": "com.github.dcendents:android-maven-gradle-plugin:1.4.1"
+]
+
+// Dependencies
+ext.deps = [
```

Comment:
Dependencies is clearly declared in a dependency block, so I'm sorry that I don't quite understand how does your change help organizing the dependencies. Also, you are renaming the dependency, each dependency is uniquely declared by groupid + artifactid + version. To me, this part is only introducing one extra level.